File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int count = 0, count2 = 0;

    for(int i = 0; i < s.size(); i++) {
        if (s[i] == 'p' || s[i] == 'P') {
            count++;
        }
        if (s[i] == 'y' || s[i] == 'Y') {
            count2++;
        }
    }
    answer = count == count2;

    return answer;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    pp = 0
    yy = 0
    for p in s:
        if p == 'p' or p == 'P':
            pp += 1
        if p == 'y' or p == 'Y':
            yy += 1
    if pp == yy:
        return True

    return False